File: ruby/fizzbuzz.rb
$i = 1
$limit = 101

while $i < $limit do
	if $i % 3 == 0 and $i % 5 == 0
		puts("fizzbuzz")
	elsif $i % 3 == 0
		puts("fizz")
	elsif $i % 5 == 0
		puts("buzz")
	else
		puts($i)
	end	
	$i += 1
end

File: scala/fizzbuzz.scala
object Main {
  def main(args: Array[String]): Unit = {
    for ( x <- 1 to 100 ){
      if ( x % 3 == 0 && x % 5 == 0) {
        print("fizzbuzz")
        println()
      } else if ( x % 3 == 0) {
        print("fizz")
        println()
      } else if ( x % 5 == 0) {
        print("buzz")
        println()
      } else {
        print(x)
        println()
      }
    }
  }
}
